test/tla/ChangRobertsTyped_Test.tla

```diff
@@ -0,0 +1,103 @@
+--------------------- MODULE ChangRobertsTyped_Test -------------------------
+(*
+ * A test setup for ChangRobertsTyped.
+ *)
+EXTENDS Integers, Apalache
```

Comment:
Why do we need to import `Apalache`?

Reply:
We are using `Gen`, which is defined in `Apalache`. If you are not using `Gen`, you would not need it.